test/unittests/capi_test.cpp

```diff
@@ -402,7 +402,7 @@ TEST(capi, get_export_description)
     fizzy_free_module(module);
 }
 
-TEST(capi, export_name_after_instantiate)
+TEST(capi_exports, export_name_after_instantiate)
```

Comment:
Shouldn't these be part of `capi_module`?

Reply:
Not sure what the classification is, but it exports a thing.

Reply:
I don't remember exactly anyomore either, but I guess I put exports-related tests into a separate file, because there're quite a few of them.
And they some of the functions tested there operate on a instance, too.